shap/plots/_beeswarm.py

```diff
@@ -421,6 +421,8 @@ def beeswarm(shap_values, max_display=10, order=Explanation.abs.mean(0),
     pl.xlabel(labels['VALUE'], fontsize=13)
     if show:
         pl.show()
+    else:
+        return pl.gcf()
```

Comment:
Suggest returning the axes:

```suggestion
        return pl.gca()
```

It would also be helpful to update the docstring to document the new return value.